peoplesoft/0.3.7/modules/ROOT/pages/index.adoc

```diff
@@ -220,17 +220,17 @@ To resolve the issue, go back to the Required dependencies panel and select the
 .. Click **Create Object manually** and click the button next to it.
 +
 [.center.text-center]
-image::ps-config-global-whitelist.png[Global Element - White List]
+image::ps-config-global-allowlist.png[Global Element - Allowlist]
```

Comment:
Make alt text something like "General properties with Enable DataSense and Create Object manually options selected."